src/api/spec/services/rpmlint_log_extractor_spec.rb

```diff
@@ -40,15 +40,15 @@
         end
       end
 
-      context 'when the _log file contains invalid byte sequences in UTF-8' do
+      context 'no _log file available and no rpmlint mark available' do
         before do
           allow(Backend::Api::BuildResults::Binaries).to receive(:file)
             .with('home:user1', 'repo1', 'arch1', 'package1', '_log')
-            .and_return(invalid_byte_sequence_in_utf8)
+            .and_return(file_fixture('rpmlint_log_extractor_log_without_mark').read)
         end
 
-        it 'extracts the summary from the build log file' do
-          expect(subject).to eq('this is an invalid byte sequence �')
+        it 'returns empty results' do
+          expect(subject).to be_nil
```

Comment:
Here we are substituting one test with another one. Could we preserve the previous test? Do you find a reason not to keep it?

Reply:
Yes we don't need this test anymore. Because with the new changes if RPM mark is not available, we just return nil. And before this change if the mark is not found we used to return the full log file. This is why we were seeing build logs in place of rpmlint.

Reply:
With the previous tests I was referring to:
```
context 'when the _log file contains invalid byte sequences in UTF-8' do
  [...]
  it 'extracts the summary from the build log file' do
```
Maybe the test should have said `'when the _log file contains the RPM mark and invalid byte sequences in UTF-8'`.

I Just want to make sure we don't face again the #15905 issue if we don't provide this test.

Reply:
I've added a new test to check invalid byte sequences without mark. And adapted an existing test for `file contains the RPM mark and invalid byte sequences` to address the cases you mentioned above